src/gutter-component.coffee

```diff
@@ -1,5 +1,5 @@
 _ = require 'underscore-plus'
-React = require 'react-atom-fork'
+React = require 'react-atom-fork/addons'
```

Comment:
This was so I could use the React.addons.classSet. I could do it without classSet, but classSet makes the code a little more compact/readable.